docs/reference/meson-compatibility.rst

```diff
@@ -26,6 +26,13 @@ versions.
    and similar installation functions. On older Meson versions, these
    arguments have no effect.
 
+.. option:: 1.2.0
+
+   Meson 1.2.0 or later is required to support the
+   ``--skip-subprojects`` option that can be passed to ``meson
+   install`` to do not include in the Python wheel files installed by
+   some or all subprojects.
```

Comment:
These last two lines have a small grammar issue ("to do not" -> "to not") and I'd put "in the Python wheel" at the end of the sentence.

Reply:
> These last two lines have a small grammar issue ("to do not" -> "to not")

This is something that somehow got stuck wired in the wrong in my brain. I write it wrong all the time...